types/block.go

```diff
@@ -7,8 +7,20 @@ type Block struct {
 	StackId             int64                       `json:"stackId"`
 	Reference           string                      `json:"reference"`
 	Name                string                      `json:"name"`
+	Shared              bool                        `json:"shared"`
```

Comment:
Would be nice to have this:
```suggestion
	IsShared              bool                        `json:"isShared"`
```